ansible/install_jenkins.yml

```diff
@@ -0,0 +1,39 @@
+- hosts: localhost
+  become: true
+  gather_facts: true
+  
+  vars:
+    jenkins_hostname: localhost
+    java_packages:
+      - java-1.8.0-openjdk
+    
+    pip_install_packages:
+      - name: docker
+
+  pre_tasks:
+  - name: update all packages 
+    yum:
+      name: '*'
+      update_only: yes
+      update_cache: yes
+      state: latest
+
+  - name: install  nano  
```

Comment:
on server, we don't need nano, the server is not used for editing purpose !

Reply:
corrected remarks

Reply:
corrected remarks